examples/lightgbm/train.py

```diff
@@ -4,18 +4,21 @@
 from sklearn.model_selection import train_test_split
 from sklearn.metrics import accuracy_score, log_loss
 import lightgbm as lgb
+import matplotlib as mpl
 
 import mlflow
 import mlflow.lightgbm
 
+mpl.use('Agg')
+
 
 def parse_args():
     parser = argparse.ArgumentParser(description='LightGBM example')
     parser.add_argument('--colsample-bytree', type=float, default=1.0,
                         help='subsample ratio of columns when constructing each tree (default: 1.0)')
     parser.add_argument('--subsample', type=float, default=1.0,
                         help='subsample ratio of the training instances (default: 1.0)')
-    parser.add_argument('--conda-env',  type=str, default=None,
+    parser.add_argument('--conda-env', type=str, default=None,
                         help='the path to a conda environment yaml file (default: None)')
```

Comment:
Let's remove this argument now that it's no longer used? Same in the xgboost example